src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Microsoft.DotNet.ApiCompatibility.csproj

```diff
@@ -15,6 +15,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" ExcludeAssets="Runtime" />
```

Comment:
Why does this one not have PrivateAssets on it?

Reply:
Oh I think maybe it should. IIRC I was hoping that dropping it here would flow to other places and mean we didn't need to change one of the other places, but that didn't work.